packages/@react-aria/dnd/src/useDragHooks.ts

```diff
@@ -0,0 +1,31 @@
+import {DraggableCollectionOptions, DraggableCollectionState, useDraggableCollectionState} from '@react-stately/dnd';
+import {useMemo} from 'react';
+
+export interface DragHooks {
+    useDraggableCollectionState(props: DraggableCollectionOptions): DraggableCollectionState
+}
+
+export type DragHookOptions = Omit<DraggableCollectionOptions, 'collection' | 'selectionManager' | 'isDragging' | 'getKeysForDrag'>
+
+export function useDragHooks(options: DragHookOptions): DragHooks {
```

Comment:
I'm wondering if this should be in a Spectrum package rather than Aria. It's kinda more specific to our Spectrum components. What do you think?

Reply:
Yeah, Spectrum does make more sense now that I think about it. Feels weird for Spectrum users to have to import something from Aria in order to use a Spectrum feature. If React Aria users like the idea, they can opt-in or make their own version of the hook.

Reply:
Just to be clear, this would be a separate spectrum package from listview right? Like a `spectrum/dnd` package right?

Reply:
Created `@react-spectrum/dnd` with `useDragHooks`